lmdeploy/vl/model/deepseek_vl2.py

```diff
@@ -0,0 +1,185 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+# adapted from https://github.com/InternLM/lmdeploy/blob/main/lmdeploy/vl/model/deepseek.py
```

Comment:
This line is unnecessary.
We only mention the adaptation if the code is ported from another repo